build.sbt

```diff
@@ -2,12 +2,13 @@ organization := "org.zalando"
 
 name := "play-etcd-watcher"
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.2"
```

Comment:
I think we have to do cross compilation instead of just bumping the version so that we could patch the library for people who still use 2.11.
@ovolynets what do you think?